web/pdf_page_view.js

```diff
@@ -320,6 +331,7 @@ var PDFPageView = (function PDFPageViewClosure() {
     draw: function PDFPageView_draw() {
       if (this.renderingState !== RenderingStates.INITIAL) {
         console.error('Must be in new state before drawing');
+        return Promise.resolve(undefined);
```

Comment:
I was simply trying to bring this code in line with the existing code in `PDFThumbnailView`, please see https://github.com/mozilla/pdf.js/blob/master/web/pdf_thumbnail_view.js#L277.
Do you want me to change both of them to e.g. `Promise.reject(...)` instead?